crates/ruma-common/src/events/room/encrypted.rs

```diff
@@ -83,9 +84,10 @@ pub enum EncryptedEventScheme {
 /// Relationship information about an encrypted event.
 ///
 /// Outside of the encrypted payload to support server aggregation.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 #[allow(clippy::manual_non_exhaustive)]
 #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+#[serde(into = "relation_serde::RelationSerHelper")]
```

Comment:
A custom `Serialize` implementation would almost certainly be more efficient since this clones the enum before serialization, but it's not super important.